packages/agoric-cli/tools/getting-started.js

```diff
@@ -105,121 +102,31 @@ export const gettingStartedWorkflowTest = async (t, options = {}) => {
-        '--dapp-template',
-        'dapp-fungible-faucet',
-        ...initOptions,
-        'dapp-foo',
-      ]),
+      await myMain(['init', ...initOptions, 'dapp-foo']),
       0,
       'init dapp-foo works',
     );
     process.chdir('dapp-foo');
 
     // ==============
-    // agoric install
-    if (process.env.AGORIC_INSTALL_OPTIONS) {
-      const opts = JSON.parse(process.env.AGORIC_INSTALL_OPTIONS);
-      installOptions.push(...opts);
-    }
-    t.is(await myMain(['install', ...installOptions]), 0, 'install works');
+    // yarn install
+    t.is(await yarn(['install']), 0, 'yarn install works');
 
     // ==============
-    // agoric start --reset
-    const startResult = makePromiseKit();
-
-    if (process.env.AGORIC_START_OPTIONS) {
-      const opts = JSON.parse(process.env.AGORIC_START_OPTIONS);
-      startOptions.push(...opts);
-    }
-
-    // TODO: Allow this to work even if the port is already used.
-    const startP = myMain(['start', '--reset', ...startOptions]);
-    finalizers.push(() => pkill(startP.childProcess, 'SIGINT'));
-
-    let stdoutStr = '';
-    if (startP.childProcess.stdout) {
-      startP.childProcess.stdout.on('data', chunk => {
-        // console.log('stdout:', chunk.toString());
-        stdoutStr += chunk.toString();
-        if (stdoutStr.match(/(^|:\s+)swingset running$/m)) {
-          startResult.resolve(true);
-        }
-      });
-    }
-    startP.childProcess.on('close', code =>
-      startResult.reject(Error(`early termination: ${code}`)),
-    );
+    // yarn start:docker
+    t.is(await yarn(['start:docker']), 0, 'yarn start:docker works');
 
-    const timeout = setTimeout(
-      startResult.reject,
-      TIMEOUT_SECONDS * 1000,
-      'timeout',
-    );
-    t.is(await startResult.promise, true, `swingset running before timeout`);
-    clearTimeout(timeout);
-
-    const openP = myMain(['open', '--no-browser'], {
-      stdio: ['ignore', 'pipe', 'inherit'],
-    });
-    t.is(await openP, 0, `open --no-browser exits successfully`);
-
-    const testDeploy = async (deployCmd, opts = {}) => {
-      const deployResult = makePromiseKit();
-      const deployP = myMain(
-        ['deploy', `--hostport=127.0.0.1:${PORT}`, ...deployCmd],
-        {
-          stdio: [opts.stdin ? 'pipe' : 'ignore', 'pipe', 'inherit'],
-        },
-      );
-
-      if (opts.stdin) {
-        // Write the input to stdin.
-        deployP.childProcess.stdin.write(opts.stdin);
-        deployP.childProcess.stdin.end();
-      }
-
-      finalizers.push(() => pkill(deployP.childProcess, 'SIGINT'));
-      const to = setTimeout(
-        deployResult.resolve,
-        TIMEOUT_SECONDS * 1000,
-        'timeout',
-      );
-      const done = await Promise.race([deployResult.promise, deployP]);
-      t.is(done, 0, `deploy ${deployCmd.join(' ')} successful before timeout`);
-      clearTimeout(to);
-    };
+    // TODO: use abci_info endpoint to get block height
+    // sleep for 180 seconds for now
```

Comment:
this comment will be stale if someone changes the constant. 
```suggestion
    // sleep to let contract start
```
("for now" is implied by the TODO)